mods/tuxemon/db/monster/aardart.json

```diff
@@ -2,6 +2,8 @@
     "slug": "aardart",
     "category": "anteater",
     "ai": "RandomAI",
+    "fusion1": "aard",
```

Comment:
Good work, but you probably should have done this in the po file.

Reply:
Why?... Do we want to be translatable?

Reply:
Do we need to remove all the fusion1/2 from here and from the wiki?